### PR TITLE
Add TRSS parser

### PR DESCRIPTION
**Pull request by karianna**

### CODE REVIEW

File: doc/Triage.md [hunk 1]
```diff
@@ -1,24 +1,41 @@
 # Test Triage at AdoptOpenJDK
 
+## TRSS Parser for Release Builds
+
+We've created a NodeJS module to parse TRSS output, convert it to Markdown so folks can paste that summary into a GitHuv issue.
```
Comment (smlambert) [inline doc/Triage.md:5]: Github versus Githuv

**Review by smlambert — COMMENTED** [review]
Minor typo, otherwise LGTM

**Review by smlambert — CHANGES_REQUESTED** [review]
A minor typo, otherwise LGTM

**Comment by smlambert** [comment]
Ping @karianna

**Review by smlambert — APPROVED** [review]
thanks @gdams and @karianna ...  I realize there are some other doc updates we should do in this file, but I'll create a separate PR.

**Review by sophia-guo — APPROVED** [review]
LGTM

**Pull request merged**